va-gov/includes/level2-index.html

```diff
@@ -11,7 +11,7 @@
   <div class="usa-grid">
     <article class="usa-width-two-thirds">
       <div class="inline hub-main-icon">
-        <i class="icon-large fa fa-{{icon}}"></i>
+        <i class="icon-large white fa fa-{{icon}} hub-background-{{icon}}"></i>
```

Comment:
This assumption of icons needing the `fa` class and icon specific classes starting with `fa-` breaks down in FA 5 (there are different prefixes for different icons) so we could use this opportunity to make it less opinionated and save ourselves some trouble down the road.